float32 shoulder_left
float32 shoulder_right
float32 elbow_left
float32 elbow_right
float32 wrist_left
float32 wrist_right
float32 wrist_rotate
float32 gripper

